modules/nw-autoscaling-ingress-controller.adoc

```diff
@@ -27,12 +30,12 @@ $ oc create serviceaccount thanos && oc describe serviceaccount thanos
 [source,terminal]
 ----
 Name:                thanos
-Namespace:           openshift-ingress-operator
+Namespace:           default
```

Comment:
This shouldn't be changed, sorry my example I sent you wasn't in the right namespacee.
```suggestion
Namespace:           openshift-ingress-operator
```

Reply:
:heavy_check_mark: